javascript-timers/timeout.js

```diff
@@ -0,0 +1,4 @@
+setTimeout(function () {
+  const $h1 = document.querySelector('.message');
```

Comment:
This line should be outside the function in the global space. By defining it here, every time this function gets called, you will search through the whole DOM tree to find the element, which is really inefficient. We just want to do the search once